content/doc/developer/internationalization/i18n-jelly-views.adoc

```diff
@@ -40,3 +40,22 @@ Properties files should be UTF-8 encoded, at least for core or plugins which alr
 More details can be found in link:https://issues.jenkins.io/browse/JENKINS-41729[JENKINS-41729].
 For older plugins, all non-ASCII characters need to be converted into hexcode.
 Modern IDEs do it automatically, so you can be just writing localizations in the target languages there.
+
+== Helpfile Extension
+
+In some objects a help section can be defined, which loads `html` files. Consider this small example:
+
+`src/main/resources/org/example/package/index.jelly`:
+[source, xml]
+----
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:s="/lib/form">
+<s:optionalBlock title="${%message}"
+                        help="/plugin/PLUGIN_NAME/help-message.html"
+                        checked="${it.config.message}"
+                        name="message-name" />
+</j:jelly>
+----
+
+The `help-message.html` file and localized versions, e.g. `help-message_de.html` for the german version, should be placed in the `webapp` folder, so that these get copied over to the plugin directory.
```

Comment:
```suggestion
The `help-message.html` file and localized versions, such as `help-message_de.html` for the german version, should be placed in the `webapp` folder, so that these get copied over to the plugin directory.
```
Something that we've been working on in documentation is updating the text to [not use e.g](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)., but if this takes away from the message then disregard.